lccs_db/__init__.py

```diff
@@ -7,8 +7,14 @@
 #
 
 """Python Land Cover Classification System Database Model."""
+import mimetypes
 
 from .ext import LCCSDatabase
 from .version import __version__
 
-__all__ = ('__version__', 'LCCSDatabase',)
\ No newline at end of file
+mimetypes.init()
+mimetypes.add_type("application/qml ", ".qml", True)
+mimetypes.add_type("application/sdl ", ".sdl", True)
```

Comment:
Take a look at the OGC recommendation below:
- https://github.com/opengeospatial/ogcapi-styles/blob/master/standard/clause_7_styles-api.adoc#requirements-class-ogc-sld-10

GeoServer doesn't seem to use these mimetypes:
- https://github.com/geoserver/geoserver/blob/e893e997434f0ca1b085896e27b1d774f8cafb6a/src/main/src/main/java/org/geoserver/catalog/SLDHandler.java#L66

My recommendation is to follow the OGC Styles-API in this case.